content/docs/gke/troubleshooting-gke.md

```diff
@@ -417,3 +417,53 @@ kubectl -n kubeflow create secret generic kubeflow-oauth \
        --from-literal=client_id=${CLIENT_ID} \
        --from-literal=client_secret=${CLIENT_SECRET}
 ```
+
+## Managed certificates can fail if service management API not enabled
```

Comment:
Would a better title be "Troubleshooting SSL Certificate Errors?

Reply:
I changed it to "Troubleshooting SSL certificate errors"